packages/server/modules/core/services/users/management.ts

```diff
@@ -269,3 +276,27 @@ export const deleteUserFactory =
 
     return await deps.deleteUserRecord(id)
   }
+
+export const changeUserRoleFactory =
+  (deps: {
+    getServerInfo: typeof getServerInfo
+    isLastAdminUser: IsLastAdminUser
+    updateUserServerRole: UpdateUserServerRole
+  }): ChangeUserRole =>
+  async ({ userId, role }) => {
+    const guestModeEnabled = (await deps.getServerInfo()).guestModeEnabled
```

Comment:
`guestModeEnabled` was a param in the old version, why is it now taken from getServerInfo?